### DISCUSSION TITLE
Skin clusters exports are supposed to work?

### DISCUSSION THREAD

**Discussion by adrdomeniconi** [Autodesk/maya-usd#3664] [Q&A] [answered]
Hi,

I'm not sure if I'm doing something not supported by the plugin (I couldn't find anything specific about that on the docs), or maybe I'm doing something wrong, but I can't export and import joints influences information.

I've created the simplest setup possible: a simple polyCube and a joint, and I've bound the cube to the joint to create a simple skin cluster. I've also put the joint under a group to not have errors when exporting and I'm exporting the mesh and the group joint.
`'file -force -options ";exportSkels=auto;exportSkin=auto;defaultUSDFormat=usda;" '
        '-typ "USD Export" -pr -es "{}";'`

I was expecting to have the joint influence stored on the USD and be able to import the file with the skincluster information, but it's not. I've tried several different setups and apporach but without success so far. What am I missing?

[test_export.zip](https://github.com/Autodesk/maya-usd/files/14607825/test_export.zip)

Thank you!

**Comment by barbalt** [comment, marked as answer]
Hello @adrdomeniconi ,
In Usd, the mesh and the skel (joints) have to be under a UsdSkelRoot. To achieve that in maya, you would need to wrap no only the joints, but also the mesh under that group that you created. In a future release, you'll be able to use the "Create new Root Prim" option of the export to create the SkelRoot without having to group the entire scene (see attached image). 
![image](https://github.com/Autodesk/maya-usd/assets/143224172/960d56c4-a25b-459e-8e09-c50b705d6519)

If you're building the plugin yourself, you can use this commit to have that working: https://github.com/Autodesk/maya-usd/commit/927b3813bf0043a60898e0d58dff317534f487d6
Otherwise, you can watch out for a future release in which that will be contained.

Regards,
